profile/README.md

```diff
@@ -0,0 +1,24 @@
+# Welcome to Keploy 👋
+
+<p style="text-align:center;" align="center">
+  <img align="center" src="/profile/logo.png" height="30%" width="30%" />
+</p>
+
+[![Slack](https://img.shields.io/badge/slack-@keyploy--slack-yellow.svg)](https://join.slack.com/t/keploy/shared_invite/zt-12rfbvc01-o54cOG0X1G6eVJTuI_orSA)
+[![Github](https://img.shields.io/badge/github-@keyploy--github-9cf.svg)](https://github.com/keploy)
+[![LinkedIN](https://img.shields.io/badge/linkedin-@keyploy--LinkedIn-lightgray.svg)](https://www.linkedin.com/company/keploy/)
+[![YouTube](https://img.shields.io/badge/youtube-@keyploy--youtube-red.svg)](https://www.youtube.com/channel/UC6OTg7F4o0WkmNtSoob34lg)
+
+[Keyploy](https://keploy.io/) is a no-code testing platform that generates tests from API calls. It converts API calls into testcases. Mocks are automatically generated with the actual request/responses.
+
+### 🏂 Haven't tried Keyploy yet? Below is a list of the most **important repositories**
+
+<br>
+
+- [Keyploy](https://github.com/keploy/keploy) : No code API testing platform. Create unit tests and data mocks from API calls.
+- [go-sdk](https://github.com/keploy/go-sdk) : Client SDK for Keploy API Platform. No Code API testing using production traffic.
+- [UI](https://github.com/keploy/ui) : UI for Keploy platform that does API testing by dependency mocking without writing any test cases.
+
```

Comment:
We also have a sample application repository for quickstarters.

Reply:
Done